diode-server/dbstore/postgres/migrations/00003_add_updated_at_triggers.sql

```diff
@@ -0,0 +1,21 @@
+-- +goose Up
+
+-- Add trigger to change_sets table
+CREATE TRIGGER update_change_sets_updated_at
+    BEFORE UPDATE ON change_sets
+    FOR EACH ROW
+    EXECUTE FUNCTION update_updated_at_column();
+
+-- Add trigger to changes table
+CREATE TRIGGER update_changes_updated_at
+    BEFORE UPDATE ON changes
+    FOR EACH ROW
+    EXECUTE FUNCTION update_updated_at_column();
```

Comment:
statements begin/end needed:

```suggestion
-- Add trigger to change_sets table
-- +goose StatementBegin
CREATE TRIGGER update_change_sets_updated_at
    BEFORE UPDATE ON change_sets
    FOR EACH ROW
    EXECUTE FUNCTION update_updated_at_column();
-- +goose StatementEnd

-- Add trigger to changes table
-- +goose StatementBegin
CREATE TRIGGER update_changes_updated_at
    BEFORE UPDATE ON changes
    FOR EACH ROW
    EXECUTE FUNCTION update_updated_at_column();
-- +goose StatementEnd
```